### ISSUE TITLE
[2.0 Beta] Unit test with mocha and node

### ISSUE THREAD

**Issue by thangngoc89** [goldfire/howler.js#422]
I'm creating a React wrapper component for howler.js 2 beta 6 ([repo](https://github.com/thangngoc89/react-howler))

But it seems like I can't require Howler with mocha

```js
// test.spec.js
var Howler = require('howler')
```

Run it with mocha (no options) got this exception:

```
ReferenceError: HowlerGlobal is not defined
    at /home/khoa/web/react-howler/node_modules/howler/howler.min.js:4:26
    at Object.<anonymous> (/home/khoa/web/react-howler/node_modules/howler/howler.min.js:4:7164)
    at Module._compile (module.js:398:26)
    at Object.Module._extensions..js (module.js:405:10)
    at Module.load (module.js:344:32)
    at Function.Module._load (module.js:301:12)
    at Module.require (module.js:354:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (/home/khoa/web/react-howler/test/index.spec.js:1:76)
    at Module._compile (module.js:398:26)
    at Object.Module._extensions..js (module.js:405:10)
    at Module.load (module.js:344:32)
    at Function.Module._load (module.js:301:12)
    at Module.require (module.js:354:17)
    at require (internal/module.js:12:17)
    at /home/khoa/web/react-howler/node_modules/mocha/lib/mocha.js:216:27
    at Array.forEach (native)
    at Mocha.loadFiles (/home/khoa/web/react-howler/node_modules/mocha/lib/mocha.js:213:14)
    at Mocha.run (/home/khoa/web/react-howler/node_modules/mocha/lib/mocha.js:453:10)
    at Object.<anonymous> (/home/khoa/web/react-howler/node_modules/mocha/bin/_mocha:393:18)
    at Module._compile (module.js:398:26)
    at Object.Module._extensions..js (module.js:405:10)
    at Module.load (module.js:344:32)
    at Function.Module._load (module.js:301:12)
    at Function.Module.runMain (module.js:430:10)
    at startup (node.js:141:18)
    at node.js:980:3
```

Tested with node 5.3 and 4.2.2

---
Update: this only happens with howler.min.js. When I require howler.core.min.js instead. Everything work perfectly

**Comment by goldfire** [comment]
Very cool! Thanks for the heads up, this should be fixed with c4d1f71dabf68e17c871b5037344d3e7547ac9f1. I'll try and get beta7 out today with that included.

**Comment by j-verre** [comment]
I am also getting this issue with beta7

// Works...
```
var Howl = require('howler/howler.core.min.js');
import {Howl, Howler} from 'howler/howler.core.min.js';

```
// ReferenceError: HowlerGlobal is not defined
```
var Howl = require('howler');
import {Howl, Howler} from 'howler';
```

**Comment by andfk** [comment]
Hi,
Im getting the exact same issue with Mocha and changing require path based on test like @thangngoc89 suggest. Maybe not the best approach?

**Comment by chrisregner** [comment]
How did you guys work around the issue? If you don't mind.

**Comment by chrisregner** [comment]
Nvm found the fix in #637.
